spec/controllers/storage_controller_spec.rb

```diff
@@ -90,7 +90,7 @@
 
         host = FactoryBot.create(:host)
         command = button.split('_', 2)[1]
-        allow_any_instance_of(Host).to receive(:supports?).with(command.to_sym).and_return(true)
+        stub_supports(Host, command.to_sym)
```

Comment:
Can stub_supports automatically call `.to_sym` under the covers or is that too much magic?